packages/react-native/Libraries/Blob/Blob.js

```diff
@@ -81,7 +81,7 @@ class Blob {
     return this._data;
   }
 
-  slice(start?: number, end?: number): Blob {
+  slice(start?: number, end?: number, contentType: string = ''): Blob {
     const BlobManager = require('./BlobManager');
     let {offset, size} = this.data;
```

Comment:
Should we inherit the previous type by default?
```suggestion
    let {offset, size, type} = this.data;
```

Reply:
@javache I'm not sure we should, the spec says to initialise this as [`null`](https://w3c.github.io/FileAPI/#slice-method-algo).  We differ from the semantics of the File Blob definition, but this is for performance reasons right?